Integrations/FeedTAXII/FeedTAXII.py

```diff
@@ -511,6 +517,31 @@ def __init__(self, insecure: bool = True, polling_timeout: int = 20, initial_int
                 self.username = credentials.get('identifier', None)
                 self.password = credentials.get('password', None)
 
+        self.all_collections = self.get_all_collections()
+        if collection is None or collection == '':
+            return_error(f"No collection set. Here is a list of all accessible collections: "
+                         f"{str(self.all_collections)}")
+
+    def get_all_collections(self):
+        """Gets a list of all collections listed in the discovery service instance.
+
+        Returns:
+            list. A list of all collection names in discovery service.
+        """
+        if self.discovery_service:
+            taxii_client = cabby.create_client(discovery_path=self.discovery_service)
+            if self.username:
+                taxii_client.set_auth(username=str(self.username), password=self.password, verify_ssl=self.verify_cert)
+            elif self.api_key:
+                taxii_client.set_auth(username=str(self.api_key), verify_ssl=self.verify_cert)
+            else:
+                taxii_client.set_auth(verify_ssl=self.verify_cert)
+
+            all_collections = taxii_client.get_collections()
```

Comment:
`get_collections` can trigger an exception, please check if the error returned by such an exception is informative when its triggered, see [the documentation](https://cabby.readthedocs.io/en/stable/api.html):
![image](https://user-images.githubusercontent.com/20818773/77311491-de99aa80-6d08-11ea-9ff2-9d21e84c8f7e.png)

Reply:
Done